### DISCUSSION TITLE
움직이는 발판 만드는법

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#410] [5. Q&A]
OnTriggerEnter( data : GameObject ){
        data.transform.parent = this.transform.parent;
    }

    OnTriggerStay(){
        this.transform.parent.Translate(Time.deltaTime,0,0);
    }

이런식으로 캐릭터의 부모를 움직이는 발판으로 설정한 후,
발판을 움직여서 캐릭터도 같이 움직이게 하려고 했습니다만
움직이지 않고 그냥 떨어지게 됩니다. 
높이를 조정하는건 제대로 딸려 올라가나 
그 외엔 제대로 작동하지 않네요

어떻게 발판이 움직일때 캐릭터도 같이 움직이게 할 수 있을까요?

**Comment by x4cf** [comment]
이곳에서 "마스터 타이머"와 "무빙큐브"를 다운로드 받아 사용하는건 어떨까요?
https://cheesafizz.notion.site/f4b997d8384347ff9922fc770bc2d864

**Reply by JeongYire** [reply]
발판이 움직이지 않는 문제가 아닙니다. 
발판이 움직일때 캐릭터도 같이 움직이지 않는 문제입니다.
제시하신 에셋은 움직이는 발판이네요, 
혹시나해서 사용해보았지만 여전히 캐릭터는 같이 움직이지 않고 가만히 서있으면 떨어집니다.

**Reply by x4cf** [reply]
제페토 캐릭터 오브젝트 이름을 "MyZepeto" 로 하시면 무빙큐브의 디택터가 작동합니다.
움직이는 발판위로 캐릭터가 올라가면 캐릭터도 큐브와 같이 이동할거에요 :)

재패토 캐릭터 오브젝트의 이름 변경 코드는 클라이언트 스타터에 있습니다

**Reply by JeongYire** [reply]
해결됐습니다

**Reply by wkdeo12** [reply]
캐릭터를 발판의 자식으로 바꿔보세요
